source/server.ts

```diff
@@ -1,24 +1,24 @@
-import cors from 'cors';
-import express from 'express';
-
-import { readFixtureSync } from './files';
-import { InputManager } from './input';
 import {
```

Comment:
Ohh, imports got messed. There are no current convention guides but in the meanwhile we shouldn't change them that much IMHO.

Reply:
Oh snap. Wasn't sure if it was the import-sort being set and no run for the project so I left it. Just reverted it :)